Src/WitsmlExplorer.Frontend/services/authorizationService.ts

```diff
@@ -29,6 +29,11 @@ export interface AuthorizationState {
   status: AuthorizationStatus;
 }
 
+export interface ConnectionInformation {
+  url: string;
```

Comment:
Can you change this field to serverUrl just to make it even more clear that this is connected to witsml server urls?

Reply:
done